frontend/admin/src/js/components/pool/EditPool/OtherRequirementsSection.tsx

```diff
@@ -82,8 +84,8 @@ export const OtherRequirementsSection = ({
     if (formValues.locationOption === LocationOption.RemoteOptional) {
       return {
         ...formValues,
-        specificLocationEn: null,
-        specificLocationFr: null,
+        specificLocationEn: undefined,
```

Comment:
I think these need to actually be null or else it won't blank out the locations when you switch it to "remote".  This leaves the databse in an inconsistent state.
![image](https://user-images.githubusercontent.com/8978655/183930050-ec8b4515-9f47-47d2-b533-107a8ea9c20e.png)

Reply:
Nice catch, update in [null location name if is remote](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3644/commits/9f80072d1de7bb36ef69e5ae38b8909ac8b4db39)